spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/TableWriter.scala

```diff
@@ -34,21 +34,13 @@ class TableWriter[T] private (
   val columnNames = rowWriter.columnNames diff writeConf.optionPlaceholders
   val columns = columnNames.map(tableDef.columnByName)
 
-  val defaultTTL = writeConf.ttl match {
```

Comment:
Thanks for the cleanup, this all looks clean to me. Once the tests pass on our jenkins (those "continuous-integration" things below i'll merge